docs/reference.md

```diff
@@ -160,6 +160,24 @@ All public methods generate by CSnakes have a built-in GIL acquisition and relea
 
 ## Exceptions
 
-CSnakes will raise a `PythonException` if an error occurs during the execution of the Python code. The `PythonException` class contains the error message from the Python interpreter.
+CSnakes will raise a `PythonInvocationException` if an error occurs during the execution of the Python code. The `PythonInvocationException` class contains the error message from the Python interpreter as the `InnerException`.

```

Comment:
That's not right is it? Isn't it `PythonRuntimeException` is the raised exception with an `InnerException` of `PythonInvocationException`

Reply:
It's correct.